index.js

```diff
@@ -0,0 +1,311 @@
+const newTask = document.getElementById('new-task');
+const addTaskButton = document.getElementById('add-task');
+const editTaskButton = document.getElementById('edit-task');
+const title = document.getElementById('inputTitle');
+const text = document.getElementById('inputText');
+const taskField = document.getElementById('currentTasks');
+const completeField = document.getElementById('completedTasks');
+const priorityFields = document.getElementsByClassName('form-check-input');
+const newSort = document.getElementById('sort-from-new');
+const oldSort = document.getElementById('sort-from-old');
+const save = document.getElementById('save');
+const switcher = document.getElementById('customSwitch1');
+
+let toDo = [];
+let completed = [];
+let index;
+let toDoCount = 0;
+let completedToDoCount = 0;
+
+
+function setTaskTime() {
+    let time = new Date().toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
+    let date = new Date().toLocaleDateString();
+    return time + ' ' + date;
+}
+
+function selectPriority() {
+    for (let i = 0; i < priorityFields.length; i++) {
+        if (priorityFields[i].checked) {
+            return priorityFields[i].value;
+        }
+    }
+}
+
+function createTaskField(el, area) {
+    let li = document.createElement('li');
+    li.classList.add("list-group-item", "d-flex", "w-100", "mb-2");
+    area.appendChild(li);
+
+    colorTask(li, el.priority);
+
+    let wrapper = document.createElement('div');
+    wrapper.classList.add("w-100", "mb-2");
+    li.appendChild(wrapper);

```

Comment:
1) You can use just a method that returns html template and pass there only parameters;  (an example of production ready template generator https://handlebarsjs.com/guide/#block-helpers for information, but you don't need to do the same. Do something very simple)
2) If you wanna do it like this so it should be 
a) refactored, 
b) methods should be created for common or close logic
c) space between methods should be added

Reply:
I think in case with preCreated HTML the better way is to do the elem.copy of preCreated task... If the task template will be changed, your task template wouldn't match it.